src/redux/saga/employer.ts

```diff
@@ -52,10 +57,20 @@ function* creatingJobPostSaga() {
     yield registerJobPostFail(e)
   }
 }
+function* deletingJobPostSaga() {
```

Comment:
This function will receive the `action` object as a parameter from `takeLatest`:
`function* deleteJobPostSaga(action: deleteJobPostAction) { const id = yield action.payload }`

Can you try this out to see if it works, this will be much cleaner for the redux store